src/renderer/marketplace/bank-accounts/list/offers-table.jsx

```diff
@@ -133,24 +134,25 @@ export const BankingOffersTable = withStyles(styles)(
 					{data.map(bank => (
 						<TableRow key={bank.id}>
 							<TableCell className={classes.flagCell}>
-								<FlagCountryName code={bank.countryCode} size="small" />
+								<FlagCountryName code={bank['Country Code']} size="small" />
 							</TableCell>
-							<TableCell className={classes.regionCell}>{bank.region}</TableCell>
+							<TableCell className={classes.regionCell}>{bank['Region']}</TableCell>
 							<TableCell className={classes.eligibilityCellBody}>
-								{bank.eligibility}
+								{bank['Eligibility'] &&
+									bank['Eligibility'].map(tag => <Tag key={tag}>{tag}</Tag>)}
 							</TableCell>
 							<TableCell className={classes.minDepositCell}>
-								{bank.minDeposit} {bank.minDepositCurrency}
+								{bank.minDeposit} {bank['Min Deposit']}
```

Comment:
minDeposit used twice here

one should be deposit currency

Reply:
This field should come formatted from airtable, no currency needed.